tests/BoltOn.Tests/Data/CosmosDb/RegistrationTask.cs

```diff
@@ -14,9 +14,10 @@ public void Run(RegistrationTaskContext context)
             {
                 context.Container.AddCosmosDb<TestSchoolCosmosDbOptions>(options =>
                 {
-                    options.Uri = "https://bolton2.documents.azure.com:443/";
-                    options.AuthorizationKey = "CJNc3RPjK3ACzRBtjOg56rJ774Y3ncyvJKCl5X2pfpMVe5wLPkr2v80pN5wWjhmZXYA0blOEsIDT4MmQifjtrg==";
-                    options.DatabaseName = "School";
+                    //fill the cosmosdb details and make sure you have collection and database exists as well inorder to run the cosmosdb integration tests .
```

Comment:
can we create the database and the container if they do not exist?

Reply:
I guess we could bt will do it in my next PR if we need that after discussing with u . Can u review this irrespective of that

Reply:
I guess the latest version has that? 

    Database database = await cosmosClient.CreateDatabaseIfNotExistsAsync(Program.DatabaseId);
    Container container = await cosmosClient.GetDatabase(Program.DatabaseId).CreateContainerIfNotExistsAsync(Program.ContainerId, "/LastName");

https://docs.microsoft.com/en-us/azure/cosmos-db/create-sql-api-dotnet-v4

Reply:
@gokulm : will do it next pr